istioctl/pkg/verifier/verifier.go

```diff
@@ -313,11 +316,14 @@ func (v *StatusVerifier) reportStatus(crdCount, istioDeploymentCount int, err er
 	v.logger.LogAndPrintf("Checked %v custom resource definitions", crdCount)
 	v.logger.LogAndPrintf("Checked %v Istio Deployments", istioDeploymentCount)
 	if istioDeploymentCount == 0 {
+		if err != nil {
+			v.logger.LogAndPrintf("! No Istio installation found: %v", err)
```

Comment:
I don't understand why we have the same message thrice?

Reply:
> Are you concerned that it logs the same message as the error return? 

Yes, do we really want to log an error here, and why not in the next steps?

Reply:
Other things LGTM.

Reply:
Thanks; I misunderstood.  Fixed the duplicate output and colorized it as well.